Warn users that Rancher in Docker isn't supported (#1166)
rancher/rancher-docs: docs/getting-started/installation-and-upgrade/other-installation-methods/rancher-on-a-single-node-with-docker/rancher-on-a-single-node-with-docker.md

description: For development and testing environments only, use a Docker install. Install Docker on a single Linux host, and deploy Rancher with a single Docker container.
---

:::caution

> Docker installs are not supported in production environments. These instructions are provided for testing and development purposes only. Please do not use this method for installing Rancher inside of production environments.

Reviewer: ```suggestion
Docker installs are not supported in production environments. These instructions are provided for testing and development purposes only. Please don't use this method to install Rancher in production environments.
```
